src/Illuminate/Support/Collection.php

```diff
@@ -1345,4 +1345,50 @@ protected function getArrayableItems($items)
 
         return (array) $items;
     }
+
+	/** Those methods exist in the latest version of the library and have been copied here */
+
+	/**
+	 * Run a dictionary map over the items.
+	 *
+	 * The callback should return an associative array with a single key/value pair.
+	 *
+	 * @param  callable  $callback
+	 * @return $this
+	 */
+	public function mapToDictionary(callable $callback)
+	{
+		$dictionary = [];
+
+		foreach ($this->items as $key => $item) {
```

Comment:
@gordonsh24 can't you use the map method instead of a loop?

Reply:
@andreasciamanna indeed, I could but I just copied it from the official library without nay modifications.